src/main/java/TransactionsApp.java

```diff
@@ -0,0 +1,17 @@
+import java.time.LocalDate;
+
+class TransactionsApp {
+    private static final TransactionDao TRANSACTION_DAO = new TransactionDao();
+
+    public static void main(String[] args) {
+        TRANSACTION_DAO.add(new Transaction(Type.INCOME, "pożyczka z banku",
```

Comment:
Jak ma to być aplikacja, która ma umożliwiać zarządzanie transakcjami, to raczej nie możemy jedynie wywołać metod w mainie. Potrzebujemy jakiś UI (czy to konsolowy - z użyciem Scannera, czy graficzny - Thymeleaf)